README.md

```diff
@@ -1,39 +1,43 @@
 <p align="center">
-  <a href="https://www.gatsbyjs.com/?utm_source=starter&utm_medium=readme&utm_campaign=minimal-starter">
-    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
-  </a>
-</p>
+    <p align="center">
+      <a href="https://github.com/process-analytics/website">
+        <img alt="Process Analytics Logo" src="public/icons/logo_64x64.ico" width="64" />
+      </a>
+    </p>
+    <p align="center"> 
+        <a href="https://github.com/process-analytics/website/releases">
+          <img alt="GitHub release (latest by date including pre-releases)" src="https://img.shields.io/github/v/release/process-analytics/website?label=changelog&include_prereleases"> 
+        </a>
+        <br>
+        <a href="CONTRIBUTING.md">
+          <img alt="PRs Welcome" src="https://img.shields.io/badge/PRs-welcome-ff69b4.svg?style=flat-square"> 
+        </a> 
+        <a href="CODE_OF_CONDUCT.md">
```

Comment:
If you click on the badge, it works, because it takes this one https://github.com/process-analytics/.github.
I take this code from the example repository 🙂